.gitmodules

```diff
@@ -16,3 +16,9 @@
 [submodule "contracts/lib/prb-math"]
 	path = contracts/lib/prb-math
 	url = https://github.com/PaulRBerg/prb-math
+[submodule "lodestar"]
+	path = lodestar
+	url = https://github.com/Snowfork/lodestar
+[submodule "go-ethereum"]
+	path = go-ethereum
+	url = https://github.com/Snowfork/go-ethereum
```

Comment:
I prefer to maintain a forked version for the reason https://github.com/Snowfork/snowbridge/pull/981#discussion_r1401401017

Btw: There is one commit in https://github.com/Snowfork/lodestar/commit/24851a96b0dfdb19b22f1ccc92a88b53cabef6ae to make beacon node work in mock mode without requiring the execution node(geth).

Reply:
We need to use geth for mock mode to deploy contracts, etc, so is this even a benefit?

Also, adding these dependencies as submodules makes our already large project even larger. Let's maybe discuss in our meeting today.

Reply:
Another reason is that last time when I upgraded lodestar to v1.12 the previous way to start the beacon node [here](https://github.com/Snowfork/snowbridge/blob/d77201bc370ab2a4da15de701871b4a643bd5b5c/web/packages/test/scripts/deploy-ethereum.sh#L49) did not even work(errors can't remember), forces me switch to [build from source](https://github.com/Snowfork/snowbridge/blob/61dadc99ac650d5c424c033121df7169effd6f61/web/packages/test/scripts/build-binary.sh#L76) which is also the recommend way for the [installation](https://chainsafe.github.io/lodestar/getting-started/installation/)

Reply:
As discussed to use submodules for now, we may switch to another beacon client implementation(e.g. Prysm) in the future.

Reply:
Yep, for lodestar. I believe we said in the call go-ethereum can stay in nixpkgs.

Reply:
I would prefer to build geth from source as well because the versions of Nixpkgs always fall behind the recent release.

e.g. I upgraded both modules to the most recent release in [commit](https://github.com/Snowfork/snowbridge/pull/981/commits/328d541c2a06c25d75e5ce4137f9d3c1f601407c) to test the Deneb hard fork on Goerli which requires geth to https://github.com/ethereum/go-ethereum/releases/tag/v1.13.10 and there is no nixpkgs available yet.